performance-timeline/po-resource-single.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>PerformanceObservers: resource</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="performanceobservers.js"></script>
+<h1>PerformanceObservers: resource</h1>
+<p>
+New resources will <a href="https://w3c.github.io/performance-timeline/#dfn-queue-a-performanceentry">queue a PerformanceEntry</a>.
+</p>
+<div id="log"></div>
+<script>
+  async_test(function (t) {
+    var img_location = document.location.origin
+       + "/performance-timeline/resources/square.png";
+    var observer = new PerformanceObserver(
+        t.step_func(function (entryList, obs) {
+          checkEntries(entryList.getEntriesByType("resource"),
+            [{ entryType: "resource", name: img_location}]);
+          observer.disconnect();
+          t.done();
+        })
+      );
+    observer.observe({entryTypes: ["resource"]});
+    var img = document.createElement("img");
+    img.src = img_location;
+    document.body.appendChild(img);
+    img = document.createElement("img");
+    img.src = img_location;
+    document.body.appendChild(img);
+  }, "Only one resource entry per resource");
```

Comment:
Oh, hmm.. Is that really the behavior we want? For example, my application fetches an image from remote server and renders it, the user scrolls and I evict that image to reduce memory pressure. Later, the user scrolls back up and I (re)inject the image once more, this time the image is sourced from the HTTP cache...

It seems like these should be separate entries? /cc @toddreifsteck